styles.css

```diff
@@ -0,0 +1,193 @@
+* {
+	box-sizing: border-box;
+    scroll-behavior: smooth;
+}
+
+body {
+    margin: 0 auto;
+    text-align: center;
+}
+
+a.anchor {
```

Comment:
not doing anything, probs want to remove the anchor class a's and put ids on the sections